Add PDF Viewer Component

YuanZhang98 opened the pull request:
I add this PDF viewer Component that allows users to display their pdf file with DataSource

Signed-off-by: Yuan Zhang <57537261+YuanZhang98@users.noreply.github.com>

melohagan commented:
Hey @YuanZhang98 

Thanks for the submission! 
<img width="487" alt="Screenshot 2022-10-20 at 18 10 46" src="https://user-images.githubusercontent.com/101575380/197014342-3a198f32-f14f-442e-8d2e-b79f0d2ecd04.png">

I'm getting an error at the moment. I may simply not be using it correctly. 
Would you mind adding some additional usage steps to your README please? I'm not quite sure why I both a `FileName` column and a `Pdf` column?

YuanZhang98 commented:
Hi @melohagan, I added some steps for the use of Data Source. The reason I added `FileName` and `Pdf` columns are because `Pdf` is using attachment type, and if I upload the pdf file it will just show PDF without the actual name of the file. That's why I added `FileName` to identify the file, the user can specify which pdf files they want to display by filtering the FileName

melohagan reviewed (approved):
Nice one, cheers for updating the README! 

LGTM! 👍
